diff --git a/dist/markdown/mdx.js b/dist/markdown/mdx.js
index c4d835a6764d3660b8b8179ae690404bdef1febc..c06e1c9530c2a45b48d990dcff60ecaaac1b1938 100644
--- a/dist/markdown/mdx.js
+++ b/dist/markdown/mdx.js
@@ -10,7 +10,7 @@ export const bundleMDX = ({ mdxString, options, contentDirPath, rawDocumentData,
     if (mdxString.length === 0) {
         return '';
     }
-    const { rehypePlugins, remarkPlugins, resolveCwd, cwd: cwd_, ...restOptions } = options ?? {};
+    const { rehypePlugins, remarkPlugins, resolveCwd, cwd: cwd_, mdxOptions: mdxOptions_, ...restOptions } = options ?? {};
     const getCwdFromContentDirPath = () => 
     // TODO don't use `process.cwd()` but instead `HasCwd`
     path.isAbsolute(contentDirPath) ? contentDirPath : path.join(process.cwd(), contentDirPath);
@@ -24,7 +24,7 @@ export const bundleMDX = ({ mdxString, options, contentDirPath, rawDocumentData,
                 ...(opts.remarkPlugins ?? []),
                 ...(remarkPlugins ?? []),
             ];
-            return opts;
+            return mdxOptions_ ? mdxOptions_(opts) : opts;
         },
         // User-provided cwd trumps resolution
         cwd: cwd_ ?? getCwd(),